kuka_rox_hardware_interface/src/robot_manager_node.cpp

```diff
@@ -126,6 +126,8 @@ RobotManagerNode::on_activate(const rclcpp_lifecycle::State &)
   if (!hw_response || !hw_response->ok) {
     RCLCPP_ERROR(get_logger(), "Could not activate hardware interface");
     // 'unset config' does not exist, safe to return
+    is_configured_msg_.data = false;
```

Comment:
Doesn't it get back to configured state? So it should still remain configured as far as I see.